File: 백준/Bronze/31994. 강당 대관/강당 대관.cs
using System;

class Q31994
{
    static void Main(string[] args)
    {
        String maxseminar = "";
        int max = 0;
        
        for (int i = 0; i < 7; i++)
        {
            string[] S = Console.ReadLine().Split(); 
            string seminar = S[0];
            int num = int.Parse(S[1]);
            
            if (num > max) {
                maxseminar = seminar;
                max = num;
            }
        }
        
        Console.WriteLine(maxseminar);
    }
}

File: 백준/Bronze/31994. 강당 대관/강당 대관.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String maxseminar = "";
        int max = 0;
        
        for (int i = 0; i < 7; i++) {
            String seminar = sc.next();
            int num = sc.nextInt();
            
            if (num > max) {
                maxseminar = seminar;
                max = num;
            }
        }
        
        System.out.println(maxseminar);
    }
}